emscripten.py

```diff
@@ -2188,7 +2188,7 @@ def load_metadata(metadata_raw):
     metadata[k] = v
 
   # Initializers call the global var version of the export, so they get the mangled name.
-  metadata['initializers'] = list(map(asmjs_mangle, metadata['initializers']))
+  metadata['initializers'] = [asmjs_mangle(str(i)) for i in metadata['initializers']]
```

Comment:
actually `asstr()` may be better than `str()` here

Reply:
Done